qiskit_ibm_runtime/fake_provider/fake_backend.py

```diff
@@ -278,20 +278,13 @@ def target(self) -> Target:
         return self._target
 
     @property
-    def max_circuits(self) -> int:
-        """(DEPRECATED) The maximum number of circuits
-
-        The maximum number of circuits that the backend supports in a single execution.
-        Note that the actual number of circuits the service allows may be different.
+    def max_circuits(self) -> None:
+        """This property used to return the `max_experiments` value from the
+        backend configuration but this value is no longer an accurate representation
+        of backend circuit limits. New fields will be added to indicate new limits.
         """
 
-        issue_deprecation_msg(
-            "max_circuits is deprecated",
-            "0.37.0",
-            "Please see our documentation on job limits "
-            "https://quantum.cloud.ibm.com/docs/guides/job-limits#job-limits.",
-        )
-        return self.configuration().max_experiments
+        return None
```

Comment:
What is the motivation to keep the property? Is it to replace it's function in the future with some other form of calculating max circuits?

Reply:
`max_circuits` is an abstract method in the [BackendV2](https://github.com/Qiskit/qiskit/blob/main/qiskit/providers/backend.py#L203) base class so I believe it was decided during deprecation that the method would just return `None`.